apps/parsley/cypress/support/commands.ts

```diff
@@ -202,13 +206,14 @@ Cypress.Commands.add(
   },
 );
 
-Cypress.Commands.add("assertValueCopiedToClipboard", (value: string) => {
-  cy.window().then((win) => {
-    win.navigator.clipboard.readText().then((text) => {
-      expect(text).to.eq(value);
-    });
-  });
-  // This wait is necessary to ensure the clipboard has time to be read
-  // eslint-disable-next-line cypress/no-unnecessary-waiting
-  cy.wait(50);
+Cypress.Commands.overwrite("visit", (originalVisit, url, options = {}) => {
+  const opts = {
+    onBeforeLoad(win: Window): void {
```

Comment:
Can we do this in a beforeEach instead or does it need to be in this onBeforeLoad hook?